src/guide/transitions-overview.md

```diff
@@ -128,17 +144,70 @@ If we were to apply these states to a transition, it would look something like t
 </p>
 <script async src="https://static.codepen.io/assets/embed/ei.js"></script>
 
+Easing can also convey the quality of material being animated. Take this pen for example, which ball do you think is hard and which is soft?
+
+<p class="codepen" data-height="500" data-theme-id="39028" data-default-tab="result" data-user="sdras" data-slug-hash="zxJWBJ" data-preview="true" style="height: 300px; box-sizing: border-box; display: flex; align-items: center; justify-content: center; border: 2px solid; margin: 1em 0; padding: 1em;" data-pen-title="Bouncing Ball Demo">
+  <span>See the Pen <a href="https://codepen.io/sdras/pen/zxJWBJ">
+  Bouncing Ball Demo</a> by Sarah Drasner (<a href="https://codepen.io/sdras">@sdras</a>)
+  on <a href="https://codepen.io">CodePen</a>.</span>
+</p>
+<script async src="https://static.codepen.io/assets/embed/ei.js"></script>
+
 You can get a lot of unique effects and make your animation very stylish by adjusting your easing. CSS allows you to modify this by adjusting a cubic bezier property, [this playground](https://cubic-bezier.com/#.17,.67,.83,.67) by Lea Verou is very helpful for exploring this.
 
 Though you can achieve great effects for simple animation with the two handles the cubic-bezier ease offers, JavaScript allows multiple handles, and therefore, allows for much more variance.
 
-TODO: add graphic
+![Ease Comparison](/images/css-vs-js-ease.svg)
+
+Take a bounce, for instance. In CSS we have to declare each keyframe, up and down. In JavaScript, we can express all of that movement within the ease, by declaring `bounce` in the GreenSock API (other JS libraries have other types of easing defaults).
+
+Here is the code used for a bounce in CSS (example from animate.css):
+
+```css
+@keyframes bounceInDown {
+  from,
+  60%,
+  75%,
+  90%,
+  to {
+    animation-timing-function: cubic-bezier(0.215, 0.61, 0.355, 1);
+  }
+
+  0% {
+    opacity: 0;
+    transform: translate3d(0, -3000px, 0) scaleY(3);
+  }
+
+  60% {
+    opacity: 1;
+    transform: translate3d(0, 25px, 0) scaleY(0.9);
+  }
+
+  75% {
+    transform: translate3d(0, -10px, 0) scaleY(0.95);
+  }
+
+  90% {
+    transform: translate3d(0, 5px, 0) scaleY(0.985);
+  }
+
+  to {
+    transform: translate3d(0, 0, 0);
+  }
+}
 
-Take this bounce, for instance. In CSS we have to declare each keyframe, up and down. In JavaScript, we can express all of that movement within the ease, by declaring `bounce` in the GreenSock API (other JS libraries have other types of easing defaults).
+.bounceInDown {
+  animation-name: bounceInDown;
+}
+```
+
+And here is the same bounce in JS using [GreenSock (GSAP)](https://greensock.com/):
```

Comment:
nitpick: I think the link to GreenSock should be added the first time we mention it instead.

Reply:
Oh, I thought I did, probably added more material, I'll fix it right up. Thanks!

Reply:
Ok all done :)